File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float c;
    do
    {
        //Prompt user for the positive input
        c = get_float("Changed owed: \n");
    }
    while (c < 0);

    //Change dollar to cent
    c = round(c * 100);

    int coins = 0;

    //Calculate the number of coins
    while (c >= 25)
    {
        c = c - 25;
        coins++;
    }
    while (c >= 10)
    {
        c = c - 10;
        coins++;
    }
    while (c >= 5)
    {
        c = c - 5;
        coins++;
    }
    while (c >= 1)
    {
        c = c - 1;
        coins++;
    }
    //Print out the amount of coins
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float
import math


# Get float value for n
while True:
    n = get_float("Change owed: ")
    if n > 0:
        break

# Turn into integer
n = int(round(n * 100))

# Assign a variable
coins = 0

# Loops for the coins
while n >= 25:
    n = n - 25
    coins = coins + 1

while n >= 10:
    n = n - 10
    coins = coins + 1

while n >= 5:
    n = n - 5
    coins = coins + 1

while n >= 1:
    n = n - 1
    coins = coins + 1

print(coins)